search-commons/src/main/java/no/unit/nva/search2/model/ResourceParameterKey.java

```diff
@@ -45,81 +46,81 @@
+    ISSN(TEXT, Constants.ENTITY_DESCRIPTION_REFERENCE_PUBLICATION_CONTEXT_ONLINE_ISSN
                  + PIPE + Constants.ENTITY_DESCRIPTION_REFERENCE_PUBLICATION_CONTEXT_PRINT_ISSN),
-    ISSN_NOT(STRING, MUST_NOT, Constants.ENTITY_DESCRIPTION_REFERENCE_PUBLICATION_CONTEXT_ONLINE_ISSN
+    ISSN_NOT(TEXT, MUST_NOT, Constants.ENTITY_DESCRIPTION_REFERENCE_PUBLICATION_CONTEXT_ONLINE_ISSN
                                + PIPE + Constants.ENTITY_DESCRIPTION_REFERENCE_PUBLICATION_CONTEXT_PRINT_ISSN),
-    ISSN_SHOULD(STRING, SHOULD, Constants.ENTITY_DESCRIPTION_REFERENCE_PUBLICATION_CONTEXT_ONLINE_ISSN
+    ISSN_SHOULD(TEXT, SHOULD, Constants.ENTITY_DESCRIPTION_REFERENCE_PUBLICATION_CONTEXT_ONLINE_ISSN
                                 + PIPE + Constants.ENTITY_DESCRIPTION_REFERENCE_PUBLICATION_CONTEXT_PRINT_ISSN),
-    ORCID(STRING, Constants.ENTITY_DESCRIPTION_CONTRIBUTORS_IDENTITY_ORC_ID),
-    ORCID_NOT(STRING, MUST_NOT, Constants.ENTITY_DESCRIPTION_CONTRIBUTORS_IDENTITY_ORC_ID),
-    ORCID_SHOULD(STRING, SHOULD, Constants.ENTITY_DESCRIPTION_CONTRIBUTORS_IDENTITY_ORC_ID),
+    ORCID(TEXT, Constants.ENTITY_DESCRIPTION_CONTRIBUTORS_IDENTITY_ORC_ID),
+    ORCID_NOT(TEXT, MUST_NOT, Constants.ENTITY_DESCRIPTION_CONTRIBUTORS_IDENTITY_ORC_ID),
+    ORCID_SHOULD(TEXT, SHOULD, Constants.ENTITY_DESCRIPTION_CONTRIBUTORS_IDENTITY_ORC_ID),
     MODIFIED_BEFORE(DATE, LESS_THAN, Constants.MODIFIED_DATE),
     MODIFIED_SINCE(DATE, GREATER_THAN_OR_EQUAL_TO, Constants.MODIFIED_DATE),
-    PROJECT(STRING, Constants.PROJECTS_ID),
-    PROJECT_NOT(STRING, MUST_NOT, Constants.PROJECTS_ID),
-    PROJECT_SHOULD(STRING, SHOULD, Constants.PROJECTS_ID),
+    PROJECT(TEXT, Constants.PROJECTS_ID),
+    PROJECT_NOT(TEXT, MUST_NOT, Constants.PROJECTS_ID),
+    PROJECT_SHOULD(TEXT, SHOULD, Constants.PROJECTS_ID),
     PUBLISHED_BEFORE(DATE, LESS_THAN, Constants.PUBLISHED_DATE),
     PUBLISHED_SINCE(DATE, GREATER_THAN_OR_EQUAL_TO, Constants.PUBLISHED_DATE),
-    TITLE(STRING, Constants.MAIN_TITLE, 2F),
-    TITLE_NOT(STRING, MUST_NOT, Constants.MAIN_TITLE),
-    TITLE_SHOULD(STRING, SHOULD, Constants.MAIN_TITLE),
-    UNIT(STRING, Constants.ENTITY_DESCRIPTION_CONTRIBUTORS_AFFILIATION_ID),
-    UNIT_NOT(STRING, MUST_NOT, Constants.ENTITY_DESCRIPTION_CONTRIBUTORS_AFFILIATION_ID),
-    UNIT_SHOULD(STRING, SHOULD, Constants.ENTITY_DESCRIPTION_CONTRIBUTORS_AFFILIATION_ID),
-    USER(STRING, Constants.RESOURCE_OWNER),
-    USER_NOT(STRING, MUST_NOT, Constants.RESOURCE_OWNER),
-    USER_SHOULD(STRING, SHOULD, Constants.RESOURCE_OWNER),
+    TITLE(TEXT, Constants.MAIN_TITLE, 2F),
+    TITLE_NOT(TEXT, MUST_NOT, Constants.MAIN_TITLE),
+    TITLE_SHOULD(TEXT, SHOULD, Constants.MAIN_TITLE),
+    UNIT(KEYWORD, Constants.ENTITY_DESCRIPTION_CONTRIBUTORS_AFFILIATION_ID),
+    UNIT_NOT(KEYWORD, MUST_NOT, Constants.ENTITY_DESCRIPTION_CONTRIBUTORS_AFFILIATION_ID),
+    UNIT_SHOULD(KEYWORD, SHOULD, Constants.ENTITY_DESCRIPTION_CONTRIBUTORS_AFFILIATION_ID),
```

Comment:
KEYWORD or TEXT ?

Reply:
Also relevant for INSTITUTION and CONTRIBUTOR

Reply:
correct!